app/Http/Requests/AddUserRequest.php

```diff
@@ -28,11 +28,36 @@ public function rules()
             'fullname' => 'required|min:6',
             'username' => 'required|min:6|unique:users,username',
             'password' => 'required|min:6',
-            'phone' => 'required|numeric|min:6',
+            'phone' => 'required|numeric|min:8',
             'birthday' => 'required|date',
             'gender' => 'required',
             'role_id' => 'required',
             'address' => 'required|min:6',
         ];
     }
+
+    public function messages()
+    {
+        return [
+            'email.required' => __('validation.required', ['attribute' => __('admin.form.email')]),
+            'fullname.required' => __('validation.required', ['attribute' => __('admin.form.fullname')]),
+            'username.required' => __('validation.required', ['attribute' => __('admin.form.username')]),
+            'username.required' => __('validation.required', ['attribute' => __('admin.form.username')]),
+            'phone.required' => __('validation.required', ['attribute' => __('admin.form.phone')]),
+            'birthday.required' => __('validation.required', ['attribute' => __('admin.form.birthday')]),
+            'gender.required' => __('validation.required', ['attribute' => __('admin.form.gender')]),
+            'role_id.required' => __('validation.required', ['attribute' => __('admin.form.role_id')]),
+            'address.required' => __('validation.required', ['attribute' => __('admin.form.address')]),
+            'birhday.date' => __('validation.date', ['attribute' => __('admin.form.birthday')]),
+            'phone.numeric' =>  __('validation.only_num', ['attribute' => __('admin.form.phone')]),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Requests/AddUserRequest.php</br>**Line**: 52</br>Expected 1 space between double arrow and "__"; 2 found